.clog.toml

```diff
@@ -0,0 +1,11 @@
+[clog]
+repository = "https://github.com/mozilla-services/autopush"
+changelog = "CHANGELOG.md"
+from-latest-tag = true
+link-style = "github"
+
+[sections]
+Refactor = ["refactor"]
+Test = ["test"]
+Doc = ["docs"]
+Chore = ["chore"]
```

Comment:
Ah, I see that "feat" and "fix" are preferred. I'm guessing these are defaulted.